network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/airtable_block.html

```diff
@@ -1 +1,10 @@
-<script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
+<div class="container {{block.block_type}}-block">
+    <div class="row justify-content-center no-gutters">
+        <div class="streamfield-content">
+            <script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe
```

Comment:
If this block is being used, and there's two or more on the same page, we should just load the javascript once. This could be a future ticket for the backlog for python to detect it and load this just once.

Reply:
to be fair: nothing uses the airtable block at the moment, and we're not entirely sure who even wanted it, so for now this should be fine.